mipidsi/examples/spi-st7789-rpi-zero-w/src/main.rs

```diff
@@ -66,15 +67,13 @@ fn main() -> ExitCode {
 
     // SPI Display
     let spi = Spi::new(Bus::Spi0, SlaveSelect::Ss1, 60_000_000_u32, Mode::Mode0).unwrap();
-    let di = SPIInterfaceNoCS::new(spi, dc);
+    let spi_device = ExclusiveDevice::new_no_delay(spi, NoCs).unwrap();
+    let di = SPIInterface::new(spi_device, dc);
     let mut delay = Delay::new();
-    let mut display = Builder::st7789(di)
-        // width and height are switched on purpose because of the orientation
-        .with_display_size(H as u16, W as u16)
-        // this orientation applies for the Display HAT Mini by Pimoroni
-        .with_orientation(mipidsi::Orientation::LandscapeInverted(true))
-        .with_invert_colors(mipidsi::ColorInversion::Inverted)
-        .init(&mut delay, None::<OutputPin>)
+    let mut display = Builder::new(ST7789, di)
+        .display_size(W as u16, H as u16)
+        .invert_colors(ColorInversion::Inverted)
+        .init(&mut delay)
```

Comment:
The orientation is probably not the same as before, but I don't own the display hat to test it.